frontend/public/components/utils/storage-class-dropdown.tsx

```diff
@@ -8,6 +8,24 @@ import { isDefaultClass } from '../storage-class';
 
 /* Component StorageClassDropdown - creates a dropdown list of storage classes */
 
+const getUnorderedItems = (items, filter:Function = ()=>true) =>
```

Comment:
Is this typing right? Maybe:
```
const getUnorderedItems = (items, filter: () => bool) =>
```
Not sure about the typing, `Function` I think is unnecessary, and `bool` vs `true`?

Reply:
Prefer the approach suggested by @TheRealJon here https://github.com/openshift/console/pull/3576/files#r359904901